sw/ground_segment/cockpit/shapes.ml

```diff
@@ -0,0 +1,112 @@
+(*
+ * This file is part of paparazzi.
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ *
+ *)
+
+ module G = MapCanvas
+ open Latlong
+ module LL = Latlong
+ open Printf
+
+
+type shdata = {
+  shid : string;
+  shlinecolor : string;
+  shfillcolor : string;
+  shtype : float;
+  shstatus : float;
+  shlatarr : float array;
+  shlonarr : float array;
+  shradius : float}
+
+type circleshape = { mutable circsh : GnoCanvas.ellipse}
```

Comment:
not really needed since you could directly add the GnoCanvas.* to the hash table